### ISSUE TITLE
Save and approve fails if transition condition is not met

### ISSUE THREAD

**Issue by janno42** [fsr-itse/EvaP#973]
If a staff member wants to approve a course by pressing "Save and approve" on a course edit page and the transition conditions are not fulfilled (e.g. because of contributors without questionnaires - see also #960) an Internal Server Error occurs.
An error message should be displayed instead.

```
Exception Type: TransitionNotAllowed at /staff/semester/.../course/.../edit
Exception Value: Transition conditions have not been met for method 'staff_approve'
```

**Comment by karyon** [comment]
will be fixed when fixing #960.

**Comment by karyon** [comment]
...lets just close this.
